### ISSUE TITLE
Bug regarding pull request size, number of lines.

### ISSUE THREAD

**Issue by Ricardo1980** [danger/danger#495]
Hello, it seems I found a bug.

I have this:
warn("This pull request is too big 🐳") if git.lines_of_code > 1500

But having this:
<img width="147" alt="screen shot 2016-08-31 at 11 22 10" src="https://cloud.githubusercontent.com/assets/738681/18125210/37bf9ac2-6f6d-11e6-901d-8c40b2a4d9a7.png">

I receive this:
<img width="684" alt="screen shot 2016-08-31 at 11 21 54" src="https://cloud.githubusercontent.com/assets/738681/18125216/3daa7b1e-6f6d-11e6-9e55-5d1c98cbbb25.png">

In other words, I have a very small PR but I receive from danger that the PR is too big.
Do you have the same problem?

**Comment by orta** [comment]
Looks like a dupe of #314 - think this is Danger somehow pulling in the last merged commit into the git diff, any chance you can leave anything you think might be useful in there?

**Comment by orta** [comment]
And this - #160
